creator-node/nginx_conf/nginx.conf

```diff
@@ -96,6 +96,22 @@ http {
             add_header X-Cache-Status $upstream_cache_status always;
         }
 
+        location /prometheus/postgres {
+            resolver 127.0.0.11 valid=30s;
+            set $upstream exporter_postgres:9187;
```

Comment:
@joaquincasares are these upstreams defined somewhere? like `exporter_postgres`?

Reply:
Yup, defined here: https://github.com/AudiusProject/audius-docker-compose/blob/main/discovery-provider/docker-compose.yml#L19

Reply:
got it. mind adding a comment about how this talks to the exporter_postgres docker container via the docker network?